#!/usr/bin/env python
PACKAGE = "velo2cam_stereoHough"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("circle_threshold", double_t, 0, "Cam circle threshold", 0.01, 0, 5)

exit(gen.generate(PACKAGE, "velo2cam_stereoHough", "Stereo"))
